build-tools-internal/src/main/java/org/elasticsearch/gradle/internal/transport/TransportVersionResourcesPlugin.java

```diff
@@ -82,16 +83,28 @@ public void apply(Project project) {
             t.into(resourceRoot + "/definitions", c -> c.from(generateManifestTask));
         });
 
+        Consumer<GenerateTransportVersionDefinitionTask> generationConfiguration = t -> {
```

Comment:
FWIW, if we made this an `Action` then we could make applying this to tasks slightly more ergonomic. Such as:
```java
Action<MyTask> commonConfig = t -> { t.setGroup("foo") };
project.getTasks.register("myTask", MyTask.class, commonConfig).configure { // unique config }
```
or
```java
project.getTasks.register("myTask", MyTask.class, t -> {
  // unique config
}.configure(commonConfig);
```

Pick your preferred style.

Reply:
I forgot about this, I'll do it the next time I tweak this file.